src/lambda/lambdaTreeDataProvider.ts

```diff
@@ -60,27 +53,6 @@ export class LambdaTreeDataProvider implements vscode.TreeDataProvider<AWSTreeNo
             callback: async () => this.refresh()
         })
 
-        registerCommand({
```

Comment:
What's the benefit of moving these commands out of lambdaTreeDataProvider? I feel it's less important to have SAM-related commands together, instead of all commands together.

Reply:
1. This is the wrong place for the commands. Only commands related to the AWS Explorer should be here.
2. As more domains are added to the extension, it is important to keep code grouped by domain. 
  * reduces coupling
  * domain groups will have different activation needs
  * domain groups may end up activated at different times

Reply:
I agree with 1 and the third part of 2. I'm thinking we should have commands grouped by when they're registered. If anything, maybe keep commands in files related to them, but have an aggregator function elsewhere that can register commands from multiple files? It'd be nice to have a high-level overview of what's registered at each point.